ntbs-service/ntbs-service.csproj

```diff
@@ -6,24 +6,24 @@
     <UserSecretsId>3736039d-9d74-48f5-9007-015e53f0b2a1</UserSecretsId>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Audit.EntityFramework.Core" Version="14.6.3"/>
```

Comment:
Not sure why all these spaces got suddenly added...

Reply:
🤷‍♂️